src/ui/src/components/core/content/CoreChatbot.vue

```diff
@@ -67,11 +67,41 @@ See the stubs for more details.
 				</div>
 			</div>
 		</template>
+		<template
+			v-if="
+				pastedImages.length > 0 &&
+				fields.enableImagePaste.value === true
+			"
+		>
+			<div class="pastedImagesArea">
+				<div class="pastedImagesList">
+					<div
+						v-for="(image, imageIndex) in pastedImages"
+						:key="imageIndex"
+						class="pastedImage"
+						:class="{
+							processing:
+								processingImages &&
+								image.includes('data:image/svg+xml'),
+						}"
+					>
+						<img :src="image" alt="Pasted image" />
```

Comment:
I think you should use `WdsSkeletonLoader` instead of a custom class

```suggestion
					>
						<WdsSkeletonLoader v-if="processingImages && image.includes('data:image/svg+xml')" />
						<img v-show="!processingImages" :src="image" alt="Pasted image" />
```